docs/azure-openshift.md

```diff
@@ -59,7 +83,12 @@ spec:
 EOF
 ```
 
-8. Check the records created for `console` routes:
+8. Check the records created for the routes:
+```bash
+$ az network dns record-set list -g "${RESOURCE_GROUP}" -z "${ZONE_NAME}" | grep console
```

Comment:
To match the removal of "console" from the instructions?
```suggestion
$ az network dns record-set list -g "${RESOURCE_GROUP}" -z "${ZONE_NAME}" 
```

Reply:
I'm not sure I understood which "removal" you were referring to.

 `az network dns record-set list -g "${RESOURCE_GROUP}" -z "${ZONE_NAME}" ` command is supposed to list all A records of a given zone. ExternalDNS (the way it's configured in this doc) will try to create A records for any route of the cluster. That's why filtering by `console` pattern using `grep`.